### ISSUE TITLE
Clarification requested on how to implement "not mapped" aria-sort=none for NSAccessibility's accessibilitySortDirection

### ISSUE THREAD

**Issue by joanmarie** [w3c/core-aam#239]
According to the spec,  [`aria-sort=none` for NSAccessibility](https://w3c.github.io/core-aam/#ariaSortNone) is "not mapped". 

But it's not clear to me how to implement that for [`accessibilitySortDirection`](https://developer.apple.com/documentation/appkit/nsaccessibility/1534962-accessibilitysortdirection), because the only available [`NSAccessibilitySortDirections`](https://developer.apple.com/documentation/appkit/nsaccessibilitysortdirection) are:

* `unknown`
* `ascending`
* `descending`

The first of those three is the expected mapping for `aria-sort=other`.  And an unknown sort order isn't the same thing as the known lack of a sort order (right?).

I suppose an implementation could just not support `accessibilitySortDirection` until the items in question become sorted. But that feels not-quite-right to me. Things that can never be sorted should of course not support `accessibilitySortDirection`. However, it seems like things which could be sorted and just happen to be unsorted should support `accessibilitySortDirection`.

Whatever the correct implementation should be, having it stated in the Core-AAM would be helpful. Guidance in the meantime would be most appreciated! Thanks in advance!
